prime-angular-frontend/src/app/modules/site-registration/pages/remote-users-page/remote-users-page.component.html

```diff
@@ -39,6 +40,21 @@
     </app-alert>
   </div>
 
+  <div class="mb-4"
+       *ngIf="formState.form.dirty && !formState.remoteUsers.controls.length">
```

Comment:
I would use a boolean like `remoteUserRemoved` for this *ngIf explicitly since there is other conditions triggering `formState.remoteUsers.controls.length` to be 0, e.g. clicking back button and choosing discard changes clears the remote users hence length becomes 0. It causes the alert flash briefly before navigating back to previous page.